kubernetes/config-maps/concent-api-worker/local_settings.py.j2

```diff
@@ -0,0 +1,32 @@
+from ..production import *
+
```

Comment:
There's no `ALLOWED_HOSTS` defined in this file. Are you sure it's going to work? I think that this setting must be defined for the `manage.py check` to pass, even if it's empty.

Reply:
We don't use it at all in `concent`. The `python manage.py check` command pass without this setting on local environment.